nano/secure/common.cpp

```diff
@@ -1133,6 +1133,12 @@ nano::election_status::election_status () :
 	handle (rsnano::rsn_election_status_create ())
 {
 }
+
+nano::election_status::election_status (rsnano::ElectionStatusHandle const * handle_a) :
+	handle (rsnano::rsn_election_status_clone (handle_a))
```

Comment:
election_status should take ownership of the handle. So instead of cloning it it should just be: ```handle (handle_a)```